packages/artillery/lib/cmds/run-fargate.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const { Command, flags } = require('@oclif/command');
+const telemetry = require('../telemetry').init();
+const { Plugin: CloudPlugin } = require('../platform/cloud/cloud');
+
+const PlatformFargateLegacy = require('@artilleryio/platform-fargate');
+const PlatformECS = require('../platform/aws-ecs/ecs');
+const { ECS_WORKER_ROLE_NAME } = require('../platform/aws/constants');
+
+class RunCommand extends Command {
+  static aliases = ['run:fargate'];
+  // Enable multiple args:
+  static strict = false;
+
+  async run() {
+    const { flags, _argv, args } = this.parse(RunCommand);
+    new CloudPlugin(null, null, { flags });
+
+    const ECS = new PlatformECS(null, null, {}, { testRunId: 'foo' });
+    await ECS.init();
+
+    flags.taskRoleName = ECS_WORKER_ROLE_NAME;
```

Comment:
Does this line imply that the documented `--task-role-name` flag is ignored?
![image](https://github.com/artilleryio/artillery/assets/64775694/7751b051-4a7b-4266-831d-80eab6ac8295)